PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -266,7 +264,9 @@ def nanoAOD_runMETfixEE2017(process,isData):
     process.nanoSequenceCommon.insert(process.nanoSequenceCommon.index(jetSequence),process.fullPatMetSequenceFixEE2017)
 
 def nanoAOD_customizeCommon(process):
-#    makePuppiesFromMiniAOD(process,True) # call this here as it calls switchOnVIDPhotonIdProducer
+    makePuppiesFromMiniAOD(process,True) # call this here as it calls switchOnVIDPhotonIdProducer
```

Comment:
Suggest to remove the comment "# call this here as it calls switchOnVIDPhotonIdProducer", as this is not true in CMSSW>=10_2_X anymore.